src/ocean_emulators/utils/loss.py

```diff
@@ -67,3 +71,83 @@ def decomposed_mse_mae(
     mae = F.l1_loss(pred, target, reduction="none")
     combined = (mse + mae) / 2
     return combined.mean(dim=(0, 2, 3))
+
+
+N_WINDOW = 25
+
+
+class MseDynamic:
+    """A loss function that scales each channel contribute equally to the loss.
```

Comment:
I'm a little confused by this sentence, is this what you mean?

```suggestion
    """A loss function that scales each channel contribution equally to the loss.
```

Reply:
I think additional description of what this does (and why it might be wanted) would be helpful.

Reply:
Expanded a bit.

Reply:
Thanks!